MergerLogic/DataTypes/S3.cs

```diff
@@ -68,10 +68,12 @@ public override void Reset()
         private List<int> GetZoomLevels()
         {
             this._logger.LogDebug($"[{MethodBase.GetCurrentMethod().Name}] start");
-            List<int> zoomLevels = new List<int>();
+            List<int> zoomLevels = new List<int>(Data<IS3Client>.MaxZoomRead);
```

Comment:
Remove the capacity, it is not nessecary.

Reply:
I dont think that it is nessecary in this case when we are dealing with small numbers.
If anything, in most cases this will be too much, we will have less zoom levels with the array capacity being 25 (currently).
This also may be a issue if the value is accidently changed to be a smaller value than the amount of zoom levels in a certain data type.